var/spack/repos/builtin/packages/visit/package.py

```diff
@@ -59,7 +183,7 @@ class Visit(CMakePackage):
 
     root_cmakelists_dir = 'src'
 
-    @when('@3.0.0:3.0.1')
+    @when('@3.0.0:3.999,develop')
```

Comment:
Do we have to run `recmake_visit.sh` when this is patched?
I get the following error:
```
1 error found in build log:
     1662    --
     1663    
     1664    Use recmake_visit.sh or search for `CMAKE_INVOKE' in CMakeCache.txt to re-run CMake with the same arguments
     1665    
     1666    
     1667    -- Configuring done
  >> 1668    CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
     1669    Please set them or make sure they are set and tested correctly in the CMake files:
     1670    OPENGL_glu_LIBRARY (ADVANCED)
     1671        linked by target "avtplotter_ser" in directory /tmp/axel/spack-stage/spack-stage-visit-develop-an2jjizo3id7bdttlm3p4ou3673zv4nh/spack-src/src/avt/Plotter
     1672        linked by target "avtplotter_par" in directory /tmp/axel/spack-stage/spack-stage-visit-develop-an2jjizo3id7bdttlm3p4ou3673zv4nh/spack-src/src/avt/Plotter
     1673        linked by target "avtqtviswindow" in directory /tmp/axel/spack-stage/spack-stage-visit-develop-an2jjizo3id7bdttlm3p4ou3673zv4nh/spack-src/src/avt/QtVisWindow
     1674        linked by target "viewersubjectproxy" in directory /tmp/axel/spack-stage/spack-stage-visit-develop-an2jjizo3id7bdttlm3p4ou3673zv4nh/spack-src/src/viewer/subjectproxy
```

Reply:
I tried hacking in a `depends_on('glu')` and this fixes this for me.